proof_of_concept/rest/serialization.py

```diff
@@ -147,18 +147,21 @@ def _deserialize_rule(user_input: JSON) -> Rule:
     """Deserialize a Rule from JSON."""
```

Comment:
Eh, all changes to this file are debris from editing. I've added a later commit which undoes the lot. Go ahead and skip this file.